chunkflow/flow.py

```diff
@@ -336,6 +341,47 @@ def normalize_section_shang_cmd(tasks, name, nominalmin, nominalmax, clipvalues)
             task['log']['timer'][name] = time() - start
         yield task
 
+@cli.command('arbitrary-chunk-op')
+@click.option('--name', type=str, default='arbitrary-chunk-op', help='name of operator.')
+@click.option('--opprogram', type=str,
+              help='python file to call.')
+@click.option('--args', type=str, default='',
+              help='args to pass in')
+@operator
+def arbitrary_chunk_op_cmd(tasks, name, opprogram, args):
+    """[operator] .
+    """
+
+    state['operators'][name] = ArbitraryChunkOperator(
+        opprogram=opprogram, args=args, name=name)
+    print('Received args for ', name, ':', args)
+    
+    for task in tasks:
+        handle_task_skip(task, name)
+        if not task['skip']:
+            start = time()
+            task['chunk'] = state['operators'][name](task['chunk'])
+            task['log']['timer'][name] = time() - start
+        yield task
+
+
+@cli.command('copy-var')
+@click.option('--name', type=str, default='copy-var', help='name of step')
+@click.option('--from-name', type=str, default='chunk',
+              help = 'Variable to be (shallow) copied/"renamed"')
+@click.option('--to-name', type=str, default='chunk',
+              help = 'New variable name')
+@operator
+def copy_var_op_cmd(tasks, name, from_name, to_name):
```

Comment:
other operator function name is just command + cmd. To be consistent with other function names, rename it to `copy_var_cmd`?